### ISSUE TITLE
[Bug]: aria-required-children 

### ISSUE THREAD

**Issue by GabyCarrizo90** [microsoft/fluentui#26851]
### Library

React v8 (@fluentui/react)

### System Info

```shell
Title: WCAG 1.3.1: Ensures elements with an ARIA role that require child roles contain them (.ms-DetailsHeader)
Tags: Accessibility, WCAG 1.3.1, aria-required-children

Issue: Ensures elements with an ARIA role that require child roles contain them (aria-required-children - https://accessibilityinsights.io/info-examples/web/aria-required-children)

Target application: Web Application

Element path: .ms-DetailsHeader

Snippet: <div role="row" class="ms-FocusZone css-271 ms-DetailsHeader root-431" data-automationid="DetailsHeader" data-focuszone-id="FocusZone246">

Related paths: 
  div[data-sizer-index="0"]
  div[data-sizer-index="1"]
  div[data-sizer-index="2"]
  div[data-sizer-index="3"]


====

This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues.
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codepen.io/GabyCarrizo90/pen/PododEY

## Actual Behavior
When run Accessibility Insights, 

This bug comes from the HTML and we don't have access to it. 

## Expected Behavior
Component shouldn't fail with [aria-required-children.]
![Accessibility bug](https://user-images.githubusercontent.com/69490382/218881129-3a894f7a-747b-4efb-b494-373a45ae2d92.jpg)

Similar issue https://github.com/microsoft/fluentui/issues/23102

### Logs

```shell
N/A
```

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by chrisdholt** [comment]
@GabyCarrizo90 Can you provide a screenshot of the dependency? I don't think the package here is the Fluent UI Web Components, this looks specifically like React code. We don't leverage focus zone or the attributes provided above.

**Comment by micahgodbolt** [comment]
Looks like details list.

**Comment by GabyCarrizo90** [comment]
If you want to reproduce the bug you can run Accessibility Insight app in the Fluent UI documentation [https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist](url) and you will get the same issue. I'm attaching a screenshot of the issue. 
![image](https://user-images.githubusercontent.com/69490382/220764762-68e4a124-f712-4cf9-b77e-a7b2683db6a0.png)

Here is a screenshot of the Accessibility Insight app check : 
![image](https://user-images.githubusercontent.com/69490382/220765369-0ae87670-88a8-4c33-8d48-d4b26bd2c631.png)

Let me know if you need anything else. Thanks!

**Comment by spmonahan** [comment]
I believe this is a false positive and the result of a bug in axe-core. See: https://github.com/dequelabs/axe-core/issues/3850
